frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1093,6 +1097,7 @@ function getInfoMenuItem(
 }
 
 function ContextMenuInner(propsWithInputRef: Props) {
+  const [lastTimeSegmentInfoWasFetched, setLastTimeSegmentInfoWasFetched] = useState(new Date());
```

Comment:
I'd rename these to `...lastTimeSegmentInfoShouldBeFetched` because when you set it, it wasn't fetched yet. Instead, you use the variable to trigger the fetch.